partner/partner-management-service/pom.xml

```diff
@@ -13,6 +13,17 @@
 	<name>partner-management-service</name>
 	
 	<dependencies>
+	   <dependency>
+			<groupId>io.mosip.kernel</groupId>
```

Comment:
why is auth-adapter added in pom?

Reply:
forgot to remove it anusha, updated it now.